content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,122 @@
+================
+Google Translate
+================
+
+*Google Translate* can be used to translate user generated text in the Odoo chatter. To translate a
+user's text from another language, click the :guilabel:`...(three dot)` icon menu to the right of
+the chatter. Then, select :guilabel:`Translate`. The content translates to the *language* set on the
+user's preferences.
+
+.. seealso::
+   :ref:`language/change-user-language`
+
+.. image:: translate/google-translate.png
+   :align: center
+   :alt: Google Translate present in an Odoo database's chatter.
+
```

Comment:
Suggesting to move this content after the **Odoo set up** section, so that the document flow follows the traditional _overview --> configuration --> using_ flow.
```suggestion
*Google Translate* can be used to translate user generated text in the Odoo chatter.
```